src/exchanges/gemini-client.js

```diff
@@ -85,13 +141,20 @@ class GeminiClient extends EventEmitter {
       this._close(this._subscriptions.get(remote_id));
       this._subscriptions.delete(remote_id);
     }
+    if (mode === 'tickers') {
+      delete this.tickersCache[market.id];
+    }
   }
 
   /** Connect to the websocket stream by constructing a path from
    * the subscribed markets.
    */
   _connect(remote_id) {
-    let wssPath = "wss://api.gemini.com/v1/marketdata/" + remote_id + "?heartbeat=true";
+    const istopOfBook = remote_id.indexOf('top_of_book') !== -1;
+    let wssPath = "wss://api.gemini.com/v1/marketdata/" + remote_id.replace(this.topOfBookIndicator, '') + "?heartbeat=true";
+    if (istopOfBook) {
+      wssPath += '&top_of_book=true';
```

Comment:
if this subscription is using the "top of  book" type, add param to full wss path